### ISSUE TITLE
Turn errors/warnings into macOS Notifications

### ISSUE THREAD

**Issue by katrinleinweber** [milke/GitFinder-Issues#33]
#32 would be much less burdensome, if there would be no pop-ups that one would need to clicked away.

**Comment by milke** [comment]
Well, the purpose of error panels is exactly what they do; get your attention and report something being really wrong. In the case of #32, it just means GitFinder apparently can't handle shallow repositories correctly.

Anyhow, I'll consider implementing this, although I personally think it'd be misleading to inform users about error using notifications.

**Comment by katrinleinweber** [comment]
You are right. I  only considered those "shallow errors", which didn't really prevent me from working, and thus seemed more like warnings to me.
